website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -1,14 +1,13 @@
 ---
-link: Migrating pre-v1.0.0 Projects
-linkTitle: Migrating pre-v1.0.0 Projects
-weight: 6
-description: Instructions for migrating a Ansible-based project built prior to v1.0.0 to use the new Kubebuilder-style layout.
+link: Migrating Projects from pre-v1.0.0 to the latest release
+linkTitle: Migrating from pre-v1.0.0 to latest
+weight: 200
+description: Instructions for migrating an Ansible-based operator built prior to `v1.0.0` to use a Kubebuilder-style.
 ---
 
 ## Overview
 
-The motivations for the new layout are related to bringing more flexibility to users and
-part of the process to [Integrating Kubebuilder and Operator SDK][integration-doc].
+The v1.0 release improves upon prior `operator-sdk` releases with a new project structure and CLI, each of which enhances project extensibility and customizability. These design changes are influenced by [`kubebuilder`](https://book.kubebuilder.io/). For further information behind this motivation, check [the differences between Kubebuilder and Operator-SDK][what-are-the-the-differences-between-kubebuilder-and-operator-sdk].
```

Comment:
Once again, I don't see a reason to link to the differences between operator-sdk and kubebuilder are here. If anything, I'd want to see a link of the similarities.

```suggestion
The v1.0 release improves upon prior `operator-sdk` releases with a new project structure and CLI, each of which enhances project extensibility and customizability. These design changes are influenced by [`kubebuilder`](https://book.kubebuilder.io/).
```